06.bowling_object/class/game.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require_relative './frame'
+require_relative './last_frame'
+require_relative './shot'
+
+class Game
+  def initialize
+    @frames = []
+  end
+
+  def play(marks)
+    shots = Shot.shot_factory(marks)
+
+    @frames = []
+    (1..10).each do |index|
+      frame = index != 10 ? Frame.new : LastFrame.new
```

Comment:
否定形の条件は読み手が一瞬「ん？」と考えてしまうので、

```suggestion
      frame = index < 10 ? Frame.new : LastFrame.new
```

とか

```suggestion
      frame = index == 10 ? LastFrame.new : Frame.new
```

みたいな条件の方がベターかもしれません

Reply:
後者で実装します！